routes/admin.js

```diff
@@ -2,8 +2,53 @@ const express = require('express');
 const router  = express.Router();
 const pollsQueries = require('../db/queries/polls');
 const choiceQueries = require('../db/queries/choices');
+const voteQueries = require('../db/queries/votes');
 
 
+<<<<<<< HEAD
+router.get('/:id',(req,res)=>{
+  const link = req.params.id.split(':')[1];
+  console.log(link);
+  pollsQueries.getPollByLink(link)
+  .then((result)=>{
+    if(result){
+      if (link == result.voter_link){
+        res.redirect(`/vote/:${link}`);
+      } else if (link == result.admin_link) {
+        const root = "localhost:8080/";
+        const pollid = result.id;
+        choiceQueries.getChoicesidandvalue(pollid)
```

Comment:
because the getChoicesandscore function was refactored, it is unneeded to call these 2 different queries, it will give the same information.

Reply:
Please look at the master version. The master needs to add the link checks and the error message parts, but I think the master version of the rest is fine.